### ISSUE TITLE
Domain openactuarial.org

### ISSUE THREAD

**Issue by franciscogarate** [actuarialopensource/website#2]
I acquired the openactuarial.org domain in 2019. I just redirect to https://github.com/actuarialopensource/awesome-actuarial

**Comment by fumitoh** [comment]
Oh, I just got openactuarial.net a few days ago and it's pointing to @wurch 's site. I wish I had known you had the domain last week!

**Comment by wurch** [comment]
Well, we can point both to the same site, but I guess it's important to establish one of them as official.

**Comment by franciscogarate** [comment]
Yes, better if both point the same site. org is (un)officially the best appreciated (like julia.org, python.org, juliactuary.org...). If necessary, I can to transfer it.

Also, I created in 2019 the following google groups: http://groups.google.com/group/openactuarial

**Comment by wurch** [comment]
I completely agree with you @franciscogarate.

We can do the same arrangement as I did with @fumitoh, it's his domain but pointing to my server.
